modules/govuk/templates/node/s_graphite/remove_old_whisper_data.erb

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+WHISPER_DIR='<%= @graphite_path %>/storage/whisper'
+
+for dir in $(ls $WHISPER_DIR); do
+  if [[ -z "$(find $WHISPER_DIR/$dir -mtime 0 )" && $? -eq 0 ]] ; then
```

Comment:
If I'm reading this bit correctly, I'd prefer a retention period of may a week or two (e.g. `-mtime 14`), so that we still have some opportunity to look at recent metrics for all machines operating recently.

Reply:
Just realised Stephen's made a similar comment :+1:

Reply:
I see this has changed now.